unitex/src/fr/umlv/unitex/process/commands/AbstractCommand.java

```diff
@@ -1,50 +1,50 @@
-/*
```

Comment:
In all the files, either the space characters (spaces->tabs) or the newline characters (lf->crlf) are being changed. So it is not clear which changes you want to merge.